primitiv/model.cc

```diff
@@ -57,10 +57,12 @@ void Model::save(const std::string &path, bool with_stats) const {
   writer << static_cast<std::uint32_t>(FileFormat::DataType::MODEL);
 
   const auto params = get_all_parameters();
+#if defined(__x86_64__) || defined(__ppc64__)
```

Comment:
`__x86_64__` and `__ppc64__` are not compiler-independent macros. It is better to define an original macro to distinguish bit lengths (in config.h.in?).